website/docs/d/linux_function_app.html.markdown

```diff
@@ -63,6 +63,8 @@ In addition to the Arguments listed above - the following Attributes are exporte
 
 * `default_hostname` - The default hostname of the Linux Function App.
 
+* `hosting_environment_id` - The hosting environment id used by Function App.
```

Comment:
The resource name in the provider is `app_servce_environment` we should make this clearer in the documentation references? Perhaps:
 ```suggestion
* `hosting_environment_id` - The ID of the App Service Environment used by this Linux Function App.
```
Can you also do similar for the other property references below?